executor/load_data.go

```diff
@@ -122,9 +110,8 @@ type commitTask struct {
 type LoadDataWorker struct {
 	UserSctx sessionctx.Context
 
-	// TODO: they should be []T to support concurrent load data.
-	encodeWorker *encodeWorker
-	commitWorker *commitWorker
+	encodeWorkers []*encodeWorker
+	commitWorkers []*commitWorker
```

Comment:
prefer we make `encode/commit` pair into a chunk worker

Reply:
I want to separate commitWorker for future use in @you06 's concurrent INSERT INTO ... SELECT FROM